test/byContract/IexecPoco/IexecPoco1.test.ts

```diff
@@ -0,0 +1,201 @@
+        beneficiary,
+        appProvider,
+        datasetProvider,
+        scheduler,
+        anyone,
+    ]: SignerWithAddress[] = [];
+    let ordersActors: OrdersActors;
+    let ordersAssets: OrdersAssets;
+    let ordersPrices: OrdersPrices;
+
+    beforeEach('Deploy', async () => {
+        // Deploy all contracts
+        proxyAddress = await loadHardhatFixtureDeployment();
+        // Initialize test environment
+        await loadFixture(initFixture);
+    });
+
+    async function initFixture() {
+        const accounts = await getIexecAccounts();
+        ({
+            iexecAdmin,
+            requester,
+            sponsor,
+            beneficiary,
+            appProvider,
+            datasetProvider,
+            scheduler,
+            anyone,
+        } = accounts);
+        iexecWrapper = new IexecWrapper(proxyAddress, accounts);
+        await iexecWrapper.setTeeBroker(ethers.constants.AddressZero);
+        ({ appAddress, datasetAddress, workerpoolAddress } = await iexecWrapper.createAssets());
+        iexecPoco = IexecInterfaceNative__factory.connect(proxyAddress, anyone);
+        iexecPocoAsRequester = iexecPoco.connect(requester);
+        ordersActors = {
+            appOwner: appProvider,
+            datasetOwner: datasetProvider,
+            workerpoolOwner: scheduler,
+            requester: requester,
+        };
+        ordersAssets = {
+            app: appAddress,
+            dataset: datasetAddress,
+            workerpool: workerpoolAddress,
+        };
+        ordersPrices = {
+            app: appPrice,
+            dataset: datasetPrice,
+            workerpool: workerpoolPrice,
+        };
+        // TODO check why this is done in 00_matchorders.js
+        // await Workerpool__factory.connect(workerpoolAddress, scheduler)
+        //     .changePolicy(35, 5)
+        //     .then((tx) => tx.wait());
+    }
+
+    // TODO
+    describe('Verify signature', () => {});
+    describe('Verify presignature', () => {});
+    describe('Verify presignature or signature', () => {});
+
+    describe('Match orders', () => {
+        it('Should match orders with all assets (Standard)', async () => {
+            const { orders } = buildOrders({
+                assets: ordersAssets,
+                requester: requester.address,
+                beneficiary: beneficiary.address,
+                tag: standardDealTag,
+                prices: ordersPrices,
+                callback: AddressZero,
+            });
+            expect(await iexecPoco.balanceOf(proxyAddress)).to.be.equal(0);
+            // Compute prices, stakes, rewards, ...
+            const dealPrice =
+                (appPrice + datasetPrice + workerpoolPrice) * // task price
+                volume;
+            const schedulerStake = await iexecWrapper.computeSchedulerDealStake(
+                workerpoolPrice,
+                volume,
+            );
+            const workerStakeByTask = await iexecWrapper.computeWorkerTaskStake(
```

Comment:
Minor
```suggestion
            const workerStakePerTask = await iexecWrapper.computeWorkerTaskStake(
```
& schedulerRewardPerTask

https://www.google.com/search?q=per+task
https://www.google.com/search?q=by+task

Reply:
Lol, was more for myself to be sure!